lib/_tls_wrap.js

```diff
@@ -1424,6 +1424,7 @@ exports.connect = function connect(...args) {
       port: options.port,
       host: options.host,
       family: options.family,
+      hints: options.hints,
```

Comment:
Why not just pass `options` instead of building a `connectOpt`? This seems fragile, next time someone adds a connect option, they will likely forget to add it here. Also, unless I go and cross-ref the connect docs, I'm not absolutely sure that _all_ connect options are handled here (as the docs now claim they are).

Reply:
Agreed, didn't do that because there was probably a reason to build `connectOpt`? Like, I don't know, conflicting options? Will check when I can.

Reply:
Passing all the options might seem unrelated to this PR, but you didn't just change the docs to add one new option, you changed them to state that all socket.connect() options are supported. If that ever becomes untrue in the future, it won't just be a missing feature, it will be a bug. This isn't a blocking comment, by any means, but I think while you are fixing this, its worth checking if a permanent fix can be made. git blame might show some relevant history.

Reply:
Yes, again I agree, will do.

Reply:
It seems this goes all the way back to https://github.com/nodejs/node/commit/af80e7bc6e6f33c582eb1f7d37c7f5bbe9f910f7. I'm still not sure why a new object was built for connect options. Perhaps it was to make between a clear distinction between IPC and TCP connection which was then adjusted in https://github.com/nodejs/node/commit/6f1caadb85f70bb902c078d9b7a96e3c4b36b31f.

Anyway it seems safe to just pass `options` as suggest. Fixed in https://github.com/nodejs/node/pull/27816/commits/2243f596dcc5b49c554701304a8c2a87e6c352de.